packages/types/src/exchange/market-data/get-candlesticks.ts

```diff
@@ -25,6 +25,7 @@ export interface ICandlestick {
   high: number;
   low: number;
   close: number;
+  volume: number;
```

Comment:
Important to mention

```suggestion
  /**
   * Volume in base currency.
   */
  volume: number;
```